src/Utils.ts

```diff
@@ -0,0 +1,54 @@
+import { RestControllerConfig, MetadataKey, Request } from "./types";
+import "reflect-metadata";
```

Comment:
Can you give me a TLDR on what we are doing here?

Reply:
"reflect-metadata" is used to extend Reflect with further functionality making it quite easy to store Metadata for a specific Class (which can later be used when the class will be instantiated). This will prevent messing around (i.e. Checking if an object is already there) with the point of object creation, which can now happen once in Application.ts

Reply:
Ah an the import is required for typescript to properly resolve the extended Reflect